data/repos.yml

```diff
@@ -481,11 +481,6 @@
   dashboard_url: false
   deploy_url: false
 
-- github_repo_name: manuals-frontend
```

Comment:
I think that, rather than removing it from this list, we add a 
`retired: true` and add a description about what it was and why it's gone
e.g https://github.com/alphagov/govuk-developer-docs/blob/c3f24e2cc7100b1d4b13c29da3cdee1548bf2d15/data/repos.yml#L59
then there is still a record and a link to the repo in the dev docs

Reply:
Thanks! Fixed.